src/Tools/dotnet-monitor/Extensibility/FileProviderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.FileProviders;
+using System.IO;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Extensibility
+{
+    public static class FileProviderExtensions
+    {
+        public static bool TryGetExtensionDefinitionPath(this IFileProvider fileProvider, string extensionName, out string definitionPath)
```

Comment:
The `extensionName` parameter isn't really just a name, it's a sub path under the file provider; consider renaming to `extensionSubPath`.

Reply:
Agreed, this is a path and not a name - I actually originally had it set to `extensionPath` but changed it when I made your suggestion https://github.com/dotnet/dotnet-monitor/pull/2829#discussion_r1015774783 - I'll switch it to `extensionSubPath`.